### DISCUSSION TITLE
Refresh the entire dashboard page, not just the graphs?

### DISCUSSION THREAD

**Discussion by jetkins** [alexjustesen/speedtest-tracker#1904] [Ideas]
I have DASHBOARD_POLLLING set to 60s, and the graphs refresh as expected, but the "Next speedtest at:" line on the admin dashboard, and the "Latest result: x minutes ago" line on the public dashboard, remain steadfastly stuck in the past and never update.  Is it possible to have the entire page refresh, not just the results?

**Comment by svenvg93** [comment]
This is a known bug and listed in https://github.com/alexjustesen/speedtest-tracker/issues/1699. Alex is on it 😄

**Reply by jetkins** [reply]
Thanks.  I also noticed today that the results page is not updating at all, despite having RESULTS_POLLING set to 60s.

**Reply by alexjustesen** [reply]
> Thanks.  I also noticed today that the results page is not updating at all, despite having RESULTS_POLLING set to 60s.

Yeah... It's broken but it'll be fixed when I release the new dash
